reporter_test.go

```diff
@@ -240,9 +241,9 @@ func TestCreateJUnitReportSchema(t *testing.T) {
 	}
 
 	// Dummy API - should not be used when there are no failed tests
-	api := newFakeReporterAPI(-1, []rainforest.RunTestDetails{})
```

Comment:
Again, don't clobber `api`. This change probably isn't required, but seems like a good idea.